.github/ISSUE_TEMPLATE/release.md

```diff
@@ -28,7 +28,7 @@ examples of each step, assuming release vX.Y.0 is being cut.
   - [ ] Confirm that all security/critical dependency update PRs from Renovate are merged into `main`
     - https://github.com/crossplane/crossplane-runtime/pulls?q=is%3Apr+is%3Aopen+label%3Aautomated
   - [ ] Created the release branch using the [GitHub UI][create-branch].
-  - [ ] (On the **Main** Branch) Created and merged an empty commit with `git commit --allow-empty -s -m "Empty commit after release-X.Y"`
```

Comment:
no more need for an empty commit to crossplane-runtime `main` because the renovate config PR will serve the same purpose (to move `main` along ahead of the tag that's in the release branch)